src/app.html

```diff
@@ -21,457 +21,470 @@
-            <div class="row">
-                <div class="col m6 l8 hide-on-small-only">
-                    <h5 class="white-text">Bio</h5>
-                    <p class="grey-text text-lighten-4">Archie has been working professionally over 16 years. He enjoys programming, playing video games on his
-                        3DS and sucking badly at Heroes of the Storm. About the only "normal" thing he likes doing is eating
-                        food.
-                    </p>
+                <div class="collection">
+                    <a title="Archimedes Trajano Resume in Word Format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.docx"><i class="fa fa-file-word-o fa-fw"> </i> Word</a>
+                    <a title="Archimedes Trajano Resume in PDF format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.pdf"><i class="fa fa-file-pdf-o fa-fw"> </i> PDF</a>
                 </div>
-                <div class="col m6 l4 s12">
-                    <h5 class="white-text">Contact me</h5>
 
-                    <p><a class="white-text waves-effect btn-flat obfuscate" href="mailto:archie...trajano...net"><i class="fa fa-fw fa-envelope-o left"></i> archie...trajano...net</a></p>
+            </div>
+        </div>
 
-                    <p>
-                        <a class="white-text waves-effect btn-flat obfuscate" href="tel:+One FourOneSix-EightFiveSix-SixSixFiveFive"><i class="fa fa-fw fa-phone left"></i> FourOneSix-EightFiveSix-SixSixFiveFive</a></p>
+        <footer class="page-footer scrollspy" id="contact">
+            <div class="container">
+                <div class="row">
+                    <div class="col m6 l8 hide-on-small-only">
+                        <h5 class="white-text">Bio</h5>
+                        <p class="grey-text text-lighten-4">Archie has been working professionally over 16 years. He enjoys programming, playing video games
+                            on his 3DS and sucking badly at Heroes of the Storm. About the only "normal" thing he likes doing
+                            is eating food.
+                        </p>
+                    </div>
+                    <div class="col m6 l4 s12">
+                        <h5 class="white-text">Contact me</h5>
 
+                        <p><a class="white-text waves-effect btn-flat obfuscate" href="mailto:archie...trajano...net"><i class="fa fa-fw fa-envelope-o left"></i> archie...trajano...net</a></p>
+
+                        <p>
+                            <a class="white-text waves-effect btn-flat obfuscate" href="tel:+One FourOneSix-EightFiveSix-SixSixFiveFive"><i class="fa fa-fw fa-phone left"></i> FourOneSix-EightFiveSix-SixSixFiveFive</a></p>
+                        <p id="chatbutton" hidden>
+                            <button class="white-text waves-effect btn-flat"><i class="fa fa-fw fa-comment left"></i> Chat with me I'm online</button></p>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)